include/sound/sof/dai-intel.h

```diff
@@ -184,8 +187,8 @@ struct sof_ipc_dai_dmic_params {
 	/* reserved for future use */
 	uint32_t reserved[5];
 
-	/**< variable number of pdm controller config */
-	struct sof_ipc_dai_dmic_pdm_ctrl pdm[0];
+	/**< Fixed number of pdm controller config */
```

Comment:
it's not really fixed, it's rather a maximum. the number of active pcm controllers is still variable, right?

Reply:
Yep, IPC is for max configuration. The active controllers count is still passed in the header and defines how much of this struct is filled. Should the comment text mention this?

Reply:
@singalsu I think so, yes

Reply:
Maybe instead add a comment on line 181 

	uint32_t num_pdm_active; /**< Number of active pdm controllers. Range is 1..SOF_DAI_INTEL_DMIC_NUM_CTRL */

Reply:
@kv2019i Yep, but I feel comment "/**< PDM controllers configuration */" is better.

Reply:
Here's new proposal version, is this part OK?